src/app/table/table.component.ts

```diff
@@ -153,4 +193,52 @@ export class TableComponent implements OnInit, OnChanges {
   private initSelection(): void {
     this.columns.add(ColumnSelectingConfig);
   }
+
+  private initSorting(): void {
+    if (this.config.sorting == null) {
```

Comment:
if you don't have any reason to use loose equality operator(==) that I am missing, than better to stick to strict equality operator(===) everywhere.
if (this.config.sorting === null)

Reply:
I usually trying to cover `null` and `undefined` cases since `null | undefined == null // true;`